src/hooks/useAdminVenues.ts

```diff
@@ -1,15 +1,8 @@
-import {
-  ReduxFirestoreQuerySetting,
-  useFirestoreConnect,
-} from "react-redux-firebase";
-
-export const adminVenuesQuery = (
-  userUid?: string
-): ReduxFirestoreQuerySetting => ({
-  collection: "venues",
-  where: [["owners", "array-contains", userUid || ""]],
-});
+import { useFirestoreConnect } from "hooks/useFirestoreConnect";
 
 export const useAdminVenues = (userUid?: string) => {
-  useFirestoreConnect(adminVenuesQuery(userUid));
+  useFirestoreConnect({
+    collection: "venues",
+    where: [["owners", "array-contains", userUid || ""]],
+  });
```

Comment:
I expect the `userUid || ""` pattern here was originally to avoid cases where `userUid` is `undefined`, but the way it is defaulted here is probably a bug/potential security issue if there was ever an empty string (`""`) in the owners array of a venue.

We should fix this so that it aligns with our current patterns, selectively connecting the query only when all of it's 'required bits' exist, and not running it (eg. making the whole query `undefined`) if not.

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/185